templates/jitsi_meet_config.js.j2

```diff
@@ -338,7 +338,7 @@ var config = {
     // If third party requests are disabled, no other server will be contacted.
     // This means avatars will be locally generated and callstats integration
     // will not function.
-    disableThirdPartyRequests: {{ jitsi_meet_disable_third_party_requests | bool | lower  }},
+    disableThirdPartyRequests: {{ not (jitsi_meet_enable_third_party_requests | bool) | lower  }},
```

Comment:
Here it should be `{{ (not jitsi_meet_enable_third_party_requests) | bool | lower  }}`, otherwise the result is `False` instead of `false` and config.json can't be parsed correctly

Reply:
Does this ``{{ (not jitsi_meet_enable_third_party_requests) | bool | lower }}`` work when the variable is a **string* like this: 
```
jitsi_meet_enable_third_party_requests: "false"
```
?

Reply:
Yes, that conditional works when the variable is of type string [ (string_type_var) return `true`, (**not** string_type_var) return `false` ]

Therefore, in the example you propose, it doesn't matter if `jitsi_meet_enable_third_party_requests: "false"` or `jitsi_meet_enable_third_party_requests: "true"` is indicated, the template will be generated without errors and always with the same result.

If your question came to the case for the backward compatibility for those who used the role previously, all the conditionals involved use variables that were renamed so they would not have defined values in their own projects.

In fact, all these changes will imply releasing a new major version of the role (v4.0.0)